src/pages/waiting-room-to-car/components/eyesight-failure-confirmation/eyesight-failure-confirmation.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'eyesight-failure-confirmation',
+  templateUrl: 'eyesight-failure-confirmation.html',
+})
+export class EyesightFailureConfirmation {
```

Comment:
The convention is to append `Component` at the end of the name of a component.

Reply:
Thanks, good spot, fixed it.